Library/Homebrew/global.rb

```diff
@@ -26,7 +26,7 @@
 end
 RUBY_BIN = RUBY_PATH.dirname
 
-HOMEBREW_USER_AGENT = "Homebrew #{HOMEBREW_VERSION} (Ruby #{RUBY_VERSION}-#{RUBY_PATCHLEVEL}; #{OS_VERSION})"
+HOMEBREW_USER_AGENT = "#{ENV["HOMEBREW_USER_AGENT"]} (Ruby #{RUBY_VERSION}-#{RUBY_PATCHLEVEL}; #{OS_VERSION})"
```

Comment:
Yeh, the brackets are a free-for-all. I think I'd probably prefer if we dropped the Ruby version in favour of `(Macintosh; Intel Mac OS X 10.9.0)` and maybe appending the curl version `curl/1.2.3` as they are the only things that are relevant and useful.